spec/javascript/components/molecules/StatusItems.test.js

```diff
@@ -1,43 +1,139 @@
 import StatusItems from 'components/molecules/StatusItems'
+import StatusItem from 'components/molecules/StatusItem'
 
-const getWrapper = (items, limit = null) => renderer.create(
+const getWrapper = (items, limit = undefined) => shallow(
   <StatusItems
     statusItems={items}
     limit={limit}
   />
 )
 
+const TIMESTAMP_AUGUST_25_2020 = 1598338800
+const TIMESTAMP_AUGUST_26_2020 = 1598400000
+
 describe('StatusItems', () => {
-  test('should render empty status items correctly', () => {
-    const wrapper = getWrapper([])
-    expect(wrapper).toMatchSnapshot()
+  describe('snapshot tests', () => {
+    test('should render empty status items correctly', () => {
+      const wrapper = getWrapper([])
+      expect(wrapper).toMatchSnapshot()
+    })
+
+    test('should render a single status item correctly', () => {
+      const wrapper = getWrapper([mockStatusItem()])
+      expect(wrapper).toMatchSnapshot()
+    })
+
+    test('should render multiple status items correctly', () => {
+      const wrapper = getWrapper(mockStatusItems())
+      expect(wrapper).toMatchSnapshot()
+    })
+
+    test('should render multiple status items with limit 0 correctly', () => {
+      const wrapper = getWrapper(mockStatusItems(), 0)
+      expect(wrapper).toMatchSnapshot()
+    })
+
+    test('should render multiple status items with limit 1 correctly', () => {
+      const wrapper = getWrapper(mockStatusItems(), 1)
+      expect(wrapper).toMatchSnapshot()
+    })
+
+    test('should render multiple status items with limit 100 correctly', () => {
+      const wrapper = getWrapper(mockStatusItems(), 100)
+      expect(wrapper).toMatchSnapshot()
+    })
   })
 
-  test('should render a single status item correctly', () => {
-    const wrapper = getWrapper([mockStatusItem()])
-    expect(wrapper).toMatchSnapshot()
+  describe('with an empty status items list', () => {
+    const statusItems = []
+
+    test('should render no status items when no limit is specified', () => {
+      expect(getWrapper(statusItems).find(StatusItem)).toHaveLength(0)
+    })
+
+    test('should render no status items when limit is 0', () => {
+      expect(getWrapper(statusItems, 0).find(StatusItem)).toHaveLength(0)
+    })
+
+    test('should render no status items when limit is 1', () => {
+      expect(getWrapper(statusItems, 1).find(StatusItem)).toHaveLength(0)
+    })
+
+    test('should render no status items when limit is 4', () => {
+      expect(getWrapper(statusItems, 4).find(StatusItem)).toHaveLength(0)
+    })
   })
 
-  test('should render multiple status items correctly', () => {
-    const wrapper = getWrapper(mockStatusItems())
-    expect(wrapper).toMatchSnapshot()
+  describe('with a single item in the status items list', () => {
+    const statusItems = [mockStatusItems()]
+
+    test('should render one status item when no limit is specified', () => {
+      expect(getWrapper(statusItems).find(StatusItem)).toHaveLength(1)
```

Comment:
`shallow` wrappers also let us do queries using components, instead of complicated selectors with only root elements and classnames.